arbitrum-docs/launch-orbit-chain/concepts/chain-ownership.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Orbit chain ownership'
+sidebar_label: 'Chain ownership'
+description: 'Overview of the technical architecture of chain ownership affordances on Orbit chains.'
+author: dzgoldman
+sme: dzgoldman
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 0
+---
+
+A **chain owner** of an <a data-quicklook-from='arbitrum-orbit'>Orbit</a> chain is an entity that can carry out critical upgrades to the chain's core protocol; this includes upgrading protocol contracts, setting core system parameters, and adding & removing other chain owners.
+
+An Orbit chain's initial chain owner is set by the chain's creator when the chain is deployed.
+
+The chain-ownership architecture is designed to give Orbit chain creators flexibility in deciding how upgrades to their chain occur.
+
+import PublicPreviewBannerPartial from '../partials/_orbit-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+### Architecture
+
+Chain ownership affordance is handled via [**Upgrade Executor**](https://github.com/OffchainLabs/upgrade-executor) contracts.
+
+Each Orbit chain is deployed with two Upgrade Executors — one on the Orbit chain itself, and one on its <a data-quicklook-from='parent-chain'>parent chain.</a> At deployment, the chain's critical affordances are given to the Upgrade Executor contracts.
+
+Some examples:
+
+- The parent chain's core protocol contracts are upgradeable proxies that are controlled by a proxy admin; the proxy admin is owned by the Upgrade Executor on the parent chain.
+- The core Rollup contract's admin role is given to the Upgrade Executor on the parent chain.
+- The affordance to call setters the ArbOwner procompile — which allows for setting system gas parameters and scheduling ArbOS ugprades (among other things) — is given to the Upgrade Executor on the Orbit chain.
+
+Calls to an Upgrade Executor can only be made by chain owners; e.g., entities granted the `EXECUTOR_ROLE` affordance on the Upgrade Executor. Upgrade executors also have the `ADMIN_ROLE` affordance granted to themselves, which lets chain owners add or remove chain owners.
+
+With this architecture, the Upgrade Executor represents a single source of truth for affordances over critical upgradability of the chain.
+
+### Upgrades
+
+Upgrades occur via a chain owner initiating a call to an Upgrade Executor, which in turns calls some chain-owned contract.
```

Comment:
I think it would be good to add some sentences after this and say that Upgrade Executor, or chain owner, here can be any kind of governance scheme, such as EOA, Multi-Signature Accounts or a DAO governance scheme. So it's based on the Orbit chain deployer's decision to choose any scheme and control the chain by that model.